frontend/src/components/admin/AuditLogsPanel.jsx

```
@@ -0,0 +1,334 @@
+    return colors[actionType] || 'bg-gray-100 text-gray-800';
+  };
+
+  const formatActionDescription = (log) => {
+    if (log.description) {
+      return log.description;
+    }
+
+    // Generate description based on action type
+    switch (log.action_type) {
+      case 'kill_switch_toggle':
+        const newState = log.new_value?.enabled ? 'activated' : 'deactivated';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)